src/createBaseForm.js

```diff
@@ -45,9 +45,9 @@ function createBaseForm(option = {}, mixins = []) {
         };
       },
 
-      componentWillReceiveProps(nextProps) {
+      componentWillReceiveProps(nextProps, nextContext) {
         if (mapPropsToFields) {
-          this.fields = mapPropsToFields(nextProps);
+          this.fields = mapPropsToFields(nextProps, nextContext);
```

Comment:
It's same issue as with **connect** from redux. You have to defined contextTypes for wrapped component. E.g.:
```
class A extends React.Component {...}

A = Form.create(mapper)(A);
A.contextTypes = {
}
```